dataland-message-queue-utils/src/main/kotlin/org/dataland/datalandmessagequeueutils/constants/QueueNames.kt

```diff
@@ -0,0 +1,9 @@
+package org.dataland.datalandmessagequeueutils.constants
+
+/**
+ * Constants to be used as queue names in rabbitmq.
+ */
+
+object QueueNames {
+    const val DATA_POINT_STORAGE = "storage.datapoint.fanout"
```

Comment:
From my 3 min research, it is recommended to name queues after their intent (e.g., `internal-storage.store_datapoints`). What do you think?

Reply:
changed